src/components/radiobutton/radiobutton.component.html

```diff
@@ -1,19 +1,20 @@
-<label [attr.for]="(id || _radioButtonId) + '-input'"
-       class="ux-radio-button"
-       [class.ux-radio-button-checked]="value === option"
-       [class.ux-radio-button-simplified]="simplified"
-       [class.ux-radio-button-disabled]="disabled"
-       [class.ux-radio-button-focused]="_focused">
-
+<label
+    [attr.for]="(id || _radioButtonId) + '-input'"
+    class="ux-radio-button"
+    [class.ux-radio-button-checked]="value === option"
+    [class.ux-radio-button-simplified]="simplified"
+    [class.ux-radio-button-disabled]="disabled"
+    [class.ux-radio-button-focused]="_focused"
+>
     <div class="ux-radio-button-container">
-
-        <input class="ux-radio-button-input"
+        <input
+            class="ux-radio-button-input"
             uxFocusIndicator
             type="radio"
             [id]="(id || _radioButtonId) + '-input'"
             [checked]="value === option"
             [disabled]="disabled"
-            [tabindex]="tabindex || value === option ? 0 : -1"
+            [tabindex]="0"
```

Comment:
Is the `tabindex` input used for anything now that it is not part of this expression?

Reply:
@donna-marijne Thanks for the feedback. Good point I will remove this. I also think explicitly setting the tabIndex here isn't required and without it the tabIndex of our radio buttons will perform as specified in the above w3 link. Only one button is focusable at the start or end depending on the direction the tab comes from with each option being selectable via the arrow keys.

Apologies on the second issue I missed it. With regards to it. I played around with this for awhile and it seems to be an issue (or correct behaviour) of all radio buttons. I checked with default html and Angular Material's implementation and they all have this same issue, which would lead me to believe this is correct behaviour.

As a workaround I added a readonly input which allows this functionality to work. Although there may be a slightly hacky solution that I could investigate further

Reply:
Well, we can't remove an input without deprecating it first and giving proper notice.

@ashley-hunter Do you remember why a tabindex input was added?

Reply:
No I don't recall why it was added. It looks like it was an attempt to add roving focus support, but I believe the browser handles that in radio groups anyway so not required.

Reply:
As discussed, let's mark the tabindex input `@deprecated` so that it can be removed in the next major release.

Reply:
If you find there is absolutely no behaviour effect of tabindex (e.g. even without the name). It would not be a breaking change to remove the input as tabindex is a property available on all html elements, so it would not cause a build to fail by removing the input. You would just check there is no unexpected behaviour change.